models/responses.go

```diff
@@ -5,4 +5,10 @@ type Responses struct {
 	List              map[string]string `json:"responses"`
 	RisklevelsVersion string            `json:"risklevels_version"`
 	QuestionsVersion  string            `json:"questions_version"`
+	Metadata          interface{}       `json:"metadata"`
+}
+
+// Path returns the main S3 path containing responses for a campaign
+func (r Responses) Path(c string) string {
+	return "responses/" + c + "/"
```

Comment:
🤷‍♂️

Reply:
The specific S3 path is embedded in the model and can be made configurable later.